src/plugins/data/common/search/search_source/fetch/get_search_params.ts

```diff
@@ -37,12 +37,15 @@ export function getSearchParamsFromRequest(
   const searchParams = getSearchParams(getConfig);
   // eslint-disable-next-line @typescript-eslint/naming-convention
   const { track_total_hits, ...body } = searchRequest.body;
+  const dataView = typeof searchRequest.index !== 'string' ? searchRequest.index : undefined;
+  const index = dataView?.title ?? `${searchRequest.index}`;
```

Comment:
It appears as though the template literal isn't needed

Reply:
`searchRequest.index` is optional and I was trying to get around the type checks while preserving the existing behavior here - that it will be treated as `"undefined"`.

It might be more useful here to actually throw some sort of error instead - I think it is a required parameter here. I'll dig into it a little more.